Kentico.Kontent.Management/ManagementClient.cs

```diff
@@ -580,6 +580,11 @@ public async Task<ContentTypeModel> ModifyContentTypeAsync(Reference identifier,
                 throw new ArgumentNullException(nameof(identifier));
             }
 
+            if (changes == null || !changes.Any())
```

Comment:
this case is not covered up with tests

Reply:
There are two tests, one for null and one for an empty list of changes.

ModifyContentTypeAsync_ChangesAreNull_ModifiesContentType
ModifyContentTypeAsync_NoChanges_ModifiesContentType